src/components/chat/chat-input.ts

```diff
@@ -0,0 +1,187 @@
+import { LitElement, html } from 'lit';
+import { property, query, state } from 'lit/decorators.js';
+import IgcIconButtonComponent from '../button/icon-button.js';
+import IgcChipComponent from '../chip/chip.js';
+import { registerComponent } from '../common/definitions/register.js';
+import IgcFileInputComponent from '../file-input/file-input.js';
+import IgcIconComponent from '../icon/icon.js';
+import { registerIconFromText } from '../icon/icon.registry.js';
+import IgcTextareaComponent from '../textarea/textarea.js';
+import { styles } from './themes/input.base.css.js';
+import {
+  type IgcMessageAttachment,
+  attachmentIcon,
+  sendButtonIcon,
+} from './types.js';
+
+/**
+ *
+ * @element igc-chat
```

Comment:
@teodosiah This change would stop the storybook confusion when rebuilding.
```suggestion
 * @element igc-chat-input
```